R/get_data.R

```diff
@@ -20,6 +20,13 @@ get_data <- function (data_string) {
     saveRDS(res, data_path)
     return(res)
   }
+  if (data_string == "housing_price"){
+    res <- clean_housing_price_data(
+      read_housing_price_data(),
+      read_municipality())
+    saveRDS(res, data_path)
+    return(res)
+  }
```

Comment:
Could you add a test for this part?

Reply:
I don't really get the mockery package, could you do it instead?

Reply:
I have tried again, can't get it done.

Reply:
I will write some tests for this

Reply:
> 
> 
> I will write some tests for this

Done